.github/workflows/ci.yml

```diff
@@ -80,3 +80,35 @@ jobs:
               env:
                   AWS_ACCESS_KEY_ID: ${{ secrets.DOCS_AWS_ACCESS_KEY_ID }}
                   AWS_SECRET_ACCESS_KEY: ${{ secrets.DOCS_AWS_SECRET_ACCESS_KEY }}
+
+    verifyCLI:
+        strategy:
+            matrix:
+                os: [macos-latest, windows-latest]
```

Comment:
Should we add `ubuntu-latest` here and remove the other CLI check from the main test?